### PR TITLE
tanidvr: init at 1.4.1

### PR DESCRIPTION
**Pull request by pho**
###### Description of changes

TaniDVR is a CLI tool for managing and capturing video from Linux-based OEM surveillance DVRs which use DVR-IP protocol (p: 37777/TCP). It has been tested with Intelbras DVRs and is expected to work with brands such as DVR365, Q-See, GWave and other.

https://sourceforge.net/projects/tanidvr/

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

I dont really know if I should put myself as the maintainer, but I did it because CONTRIBUTING.md required a maintainer to be filled in. Is this correct?

### CODE REVIEW

File: pkgs/applications/video/tanidvr/default.nix [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  version = "1.4.1";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/tanidvr/TaniDVR/${pname}-${version}/${pname}-${version}.tar.bz2";
+    sha256 = "0irwwf6mb72n3y4xcrl3s081nbnldvdlc6ypjqxa4p32c1d0g6ql";
+  };
+
+  meta = {
+    description = "TaniDVR is a CLI tool for managing and capturing video from Linux-based OEM surveillance DVRs which use DVR-IP protocol";
+    homepage = "https://sourceforge.net/projects/tanidvr";
```
Comment (emilylange) [inline pkgs/applications/video/tanidvr/default.nix:14]:
```suggestion
    homepage = "https://tanidvr.sourceforge.net";
```

File: pkgs/applications/video/tanidvr/default.nix [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    license = lib.licenses.gpl3;
+    maintainers = [ lib.maintainers.pho ];
```
Comment (emilylange) [inline pkgs/applications/video/tanidvr/default.nix:16]: Nit, feel free to ignore:

```suggestion
    maintainers = with lib.maintainers; [ pho ];
```

File: maintainers/maintainer-list.nix
Comment (emilylange) [inline maintainers/maintainer-list.nix:1]: Please split this change into its own commit called `maintainers: add pho`.
See https://nixos.org/manual/nixpkgs/unstable/#var-meta-maintainers :)

So two commits in total:
1. `maintainers: add pho`
1. `tanidvr: init at 1.4.1`

**Review by emilylange — CHANGES_REQUESTED** [review]
Hi! Welcome to nixpkgs :)

Just a few suggestions. Please don't hesitate to ask questions if anything is unclear

File: maintainers/maintainer-list.nix [hunk 3]
```diff
@@ -11731,6 +11731,11 @@
     githubId = 421510;
     name = "Noé Rubinstein";
   };
+  pho = {
+    github = "pho";
+    githubId = 88469;
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:11736]: Maybe think about adding another mean of contact.

**Comment by pho** [comment]
Uhm, I somehow messed up the PR. I'll try to fix it without opening a new one
Thanks for the feedback!

File: pkgs/applications/video/tanidvr/default.nix [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+  src = fetchurl {
+    url = "mirror://sourceforge/tanidvr/TaniDVR/${pname}-${version}/${pname}-${version}.tar.bz2";
+    sha256 = "0irwwf6mb72n3y4xcrl3s081nbnldvdlc6ypjqxa4p32c1d0g6ql";
+  };
+
+  meta = {
+    description = "CLI tool for managing and capturing video from DVRs which use the DVR-IP protocol";
+    homepage = "https://tanidvr.sourceforge.net/";
+    license = lib.licenses.gpl3;
```
Comment (emilylange) [inline pkgs/applications/video/tanidvr/default.nix:15]:
```suggestion
    license = lib.licenses.gpl3Only;
```

**Review by emilylange — CHANGES_REQUESTED** [review]

**Review by emilylange — APPROVED** [review]
